Sources/AppStoreConnectCLI/Commands/Profiles/AppStoreConnectService+ProfileId.swift

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Foundation
+import Combine
+
+extension AppStoreConnectService {
+
+    enum ProfileError: Error, LocalizedError {
+        case notFound(String)
+        case notUnique(String)
+
+        var failureReason: String? {
```

Comment:
We might want to use this in order to print error message in `run() throws`
```suggestion
        var errorDescription: String? {
```

Reply:
Good point.